ng5/src/app/screens/register/register.component.ts

```diff
@@ -37,6 +40,7 @@ export class RegisterComponent {
                       password: post.password,
                       password_confirmation: post.password
                       }
-    console.log({user});
+    this.userService = new UserService();
```

Comment:
you shouldn't create new instances of services. the services are provided and injected via "dependency injection".
maybe this will clarify how to do it: https://angular.io/tutorial/toh-pt4